distro_test.go

```diff
@@ -163,6 +175,59 @@ configuration: error in GetConfiguration: failed to convert %q to UTF16
 	}
 }
 
+// The subtests can be parallel but the main body cannot, since it registers a
+// distro, possibly interfeering with other tests.
+//
+//nolint:tparallel
+func TestGUID(t *testing.T) {
+	// This test validates that the GUID is properly obtained and printed.
+	// Note that windows.GUID has a String method printing the expected
+	// format "{XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX}", but syscall.GUID
+	// does not have such method and prints its contents like any other
+	// struct.
+
+	realDistro := wsl.NewDistro(uniqueDistroName(t))
+	fakeDistro := wsl.NewDistro(uniqueDistroName(t))
+	wrongDistro := wsl.NewDistro(uniqueDistroName(t) + "\x00invalidcharacter")
+
+	err := realDistro.Register(emptyRootFs)
+	require.NoError(t, err, "could not register empty distro")
+	t.Cleanup(func() { realDistro.Unregister() }) //nolint:errcheck // We don't care about cleanup errors
```

Comment:
yeah, the lint is inline which always make is heard to read IMHO.

Reply:
Fair enough, I moved it to its own line.